pepper-apis/docs/specification/src/components/request-bodies.yml

```diff
@@ -65,6 +65,20 @@ RegisterRequestBody:
           invitationId:
             type: string
             description: if present, server will associate the presumably account-less user with the auth0UserId after verifying the invitation
+          firstName:
+            type: string
+            description: user's first name, if available
+          lastName:
+            type: string
+            description: user's last name, if available
+          languageCode:
+            type: string
+            description: user's preferred language, as an ISO code
```

Comment:
This needs additional details. What type of ISO code is allowed here? Is it `ISO 639-1`? Are `ISO 639-2` and `ISO 639-3` codes allowed as well? Are we using some other ISO standard?